parser.py

```diff
@@ -15,23 +15,20 @@
 upperCase = False
 
 def read_val(map_dict, key):
-    try:
-        return map_dict[key]
-    except:
-        return ''
+    return map_dict.get(key,'')
```

Comment:
We can remove this method entirely and use the dictionary get logic inline.

Reply:
Can you elucidate ?

Reply:
I meant we can remove the `read_val` method entirely and call the dictionary `.get` method wherever `read_val` is used.